tests/ops/functions/test_iterative_qpe.py

```diff
@@ -23,15 +23,15 @@
 class TestIQPE:
     """Test to check that the iterative quantum phase estimation function works as expected."""
 
+    @pytest.mark.parametrize("mcm_method", ["deferred", "tree-traversal"])
     @pytest.mark.parametrize("phi", (1.0, 2.0, 3.0))
-    def test_compare_qpe(self, phi):
+    def test_compare_qpe(self, mcm_method, phi):
         """Test to check that the results obtained are equivalent to those of QuantumPhaseEstimation"""
 
         # TODO: When we have general statistics on measurements we can calculate it exactly with qml.probs
         dev = qml.device("default.qubit", shots=10000000)
```

Comment:
That's a lot of shots 👀

Reply:
Yes, not possible with `one-shot` really.